### PR TITLE
ge(ch02s08): Added plain Java producer with plain viewer

### PR DESCRIPTION
**Pull request by psolarvi**
This plain Java producer also includes a viewer.. which is a consumer.

The paths are not final and open to discussion.

### CODE REVIEW

**Review by mabulgu — COMMENTED** [review]
Should we add another folder near `solutions`? Because it seems weird a bit with the `plain` in the same level. So we can have `plain` folders in same level but different directories. WDTY? Sth like `exercise` or sth like this would be fine. We can thiscuss this together cc @jramcast

**Review by mabulgu — COMMENTED** [review]
LGTM. I have a few comments on folder structure and kubefiles (and generated templates).

File: kafka-producers/solutions/kubefiles/create-topic.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
```
[the remark this answers is not in the document]
Comment (jramcast) [reply kafka-producers/solutions/kubefiles/create-topic.yaml:1]: Yeah, for the next iteration, I think these files make more sense in `kafka-producers/kubefiles`

**Review by jramcast — APPROVED** [review]
A nice start for an MVP.

As @mabulgu mentioned, we may want to rethink the structure, but I would do that in the next iteration. In any case, it looks ok to me as it is now

**Pull request merged**